docs/modules/user-guide/partials/proc_creating-a-devfile-stack.adoc

```diff
@@ -0,0 +1,18 @@
+[id="creating-a-devfile-stack_{context}"]
+= Creating a Devfile stack
+
+[role="_abstract"]
+Create a devfile stack to package into the devfile registry.
```

Comment:
Can we include a value or purpose behind creating a devfile stack to package into the registry? 

Like, `Create a devfile stack to package into the devfile registry so you can <state value.>` 

Would this work?:

`Create a devfile stack to package into the devfile registry so you can quickly access, share, and duplicate your different stacks for your various projects.`